server/notification.js

```diff
@@ -86,38 +86,74 @@ class Notification {
             try {
                 // If heartbeatJSON is null, assume we're testing.
                 if (heartbeatJSON == null) {
-                    let data = {
-                        username: "Uptime-Kuma",
+                    let discordtestdata = {
+                        username: "Uptime Kuma",
                         content: msg,
                     }
-                    await axios.post(notification.discordWebhookUrl, data)
+                    await axios.post(notification.discordWebhookUrl, discordtestdata)
                     return okMsg;
                 }
                 // If heartbeatJSON is not null, we go into the normal alerting loop.
                 if (heartbeatJSON["status"] == 0) {
-                    var alertColor = "16711680";
-                } else if (heartbeatJSON["status"] == 1) {
-                    var alertColor = "65280";
+                    let discorddowndata = {
+                    username: "Uptime Kuma",
+                    embeds: [{
+                        title: "❌ One of your services went down. ❌",
+                        color: 16711680,
+                        timestamp: heartbeatJSON["time"],
+                        fields: [
+                            {
+                                name: "Service Name",
+                                value: monitorJSON["name"],
+                            },
+                            {
+                                name: "Service URL",
+                                value: monitorJSON["url"],
+                            },
+                            {
+                                name: "Time (UTC)",
+                                value: heartbeatJSON["time"],
+                            },
+                            {
+                                name: "Error",
+                                value: heartbeatJSON["msg"],
+                            },
+                        ],
+                    }],
                 }
-                let data = {
-                    username: "Uptime-Kuma",
+                await axios.post(notification.discordWebhookUrl, discorddowndata)
+                return okMsg;
+                    
+                } else if (heartbeatJSON["status"] == 1) {
+                    let discordupdata = {
+                    username: "Uptime Kuma",
                     embeds: [{
-                        title: "Uptime-Kuma Alert",
-                        color: alertColor,
+                        title: "✅ Your service " +  monitorJSON["name"] + " is up! ✅",
+                        color: 65280,
```

Comment:
Would recommend replacing these with globals, like `color: colors.green`, same with the alert color.

Reply:
Hi! Isn't that for discordjs?. This is just a webhook payload. See here: https://leovoel.github.io/embed-visualizer/